comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph.md

```diff
@@ -36,51 +36,4 @@ A *path* is **simple** if it contains each edge and vertex at most *once*. If a
 
 Given an edge between two vertices *u* and *v*, we could say that *u* **reaches** *v* and *v* is **reachable** from *u*. In case of an undirected edge, the converse is also true: *v* **reaches** *u* and *u* is **reachable** from *v*.
 
-A graph is **connected** if between every vertex in the graph, there is a *path*. The graph above was connected, while the one below isn't (you cannot get from *A* to *B*, for example):
-
-![notconnected](https://img.enkipro.com/e59ce9c8052bc0aed3ef2ec9db706c1d.png)
-
-A **directed graph** is **strongly connected** if for any two vertices, *v* reaches *u* and *u* reaches *v*. In case either one of `(B,E)` or `(E,B)` edges wasn't there, *B* wouldn't have been part of the strongly connected component anymore.
-
-![strongly](https://img.enkipro.com/c5b0e80a6b40292bae6d30017577af51.png)
-
-A **subgraph** of a graph G is a graph H whose vertices and edges are a subset of the ones in G. If H contains all vertices in G, it is called a **spanning subgraph**.
-
-![spanning](https://img.enkipro.com/806d2b85d517b7e00e2f1d5fcb670173.png)
-
-If a graph is not connected, the maximal connected subgraphs are called **connected components** of the graph. The only illustration of an unconnected graph above has two connected components.
-
-A **forest** is a simple graph without cycles:
-
-![forest](https://img.enkipro.com/89e3f6008d370b4091aaf1dd10ae9af6.png)
-
-If the graph is connected as well, it is called a **tree**. A **spanning tree** is a *spanning subgraph* (a subgraph containing all of the graph's nodes) that is a tree. As the tree below contains all nodes in the initial graph, it is a spanning tree:
-
-![spanningtree](https://img.enkipro.com/48d9282e1686cda71848ec61545ed099.png)
-
-
-Note: The notion of **tree** here is not related to the one of the *Tree Data Type* since it *does not need* to have a root.
-
----
-## Practice
```

Comment:
we could add a  here like this:

What is a cycle?

???

* good answer
* bad answer